src/ui/component/header/view.jsx

```diff
@@ -53,54 +70,55 @@ const Header = (props: Props) => {
         <WunderBar />
 
         <div className="header__navigation">
-          <Button
-            className="header__navigation-item header__navigation-item--right-action"
-            activeClass="header__navigation-item--active"
-            label={
-              roundedBalance > 0 ? (
+          <Menu>
+            <MenuButton className="header__navigation-item menu__title">
+              <Icon icon={ICONS.ACCOUNT} />
+              {roundedBalance > 0 ? (
                 <React.Fragment>
                   {roundedBalance} <LbcSymbol />
                 </React.Fragment>
               ) : (
                 __('Account')
-              )
-            }
-            icon={ICONS.ACCOUNT}
-            navigate="/$/account"
-          />
-
-          <Button
-            className="header__navigation-item header__navigation-item--right-action"
-            activeClass="header__navigation-item--active"
-            description={__('Publish content')}
-            icon={ICONS.UPLOAD}
-            iconSize={24}
-            navigate="/$/publish"
-          />
-
-          {/* @if TARGET='app' */}
-          {showUpgradeButton && (
-            <Button
-              className="header__navigation-item header__navigation-item--right-action header__navigation-item--upgrade"
-              icon={ICONS.DOWNLOAD}
-              iconSize={24}
-              label={__('Upgrade App')}
-              onClick={downloadUpgradeRequested}
-            />
-          )}
-          {/* @endif */}
-
-          <Button
-            className="header__navigation-item header__navigation-item--right-action"
-            activeClass="header__navigation-item--active"
-            icon={ICONS.SETTINGS}
-            iconSize={24}
-            navigate="/$/settings"
-          />
+              )}
+            </MenuButton>
+            <MenuList>
+              <MenuItem className="menu__link" onSelect={() => history.push(`/$/account`)}>
+                <Icon aria-hidden icon={ICONS.OVERVIEW} />
+                {__('Overview')}
+              </MenuItem>
+              <MenuItem className="menu__link" onSelect={() => history.push(`/$/wallet`)}>
+                <Icon aria-hidden icon={ICONS.WALLET} />
+                {__('Wallet')}
+              </MenuItem>
+              <MenuItem className="menu__link" onSelect={() => history.push(`/$/publish`)}>
+                <Icon aria-hidden icon={ICONS.UPLOAD} />
+                {__('Publish')}
+              </MenuItem>
+            </MenuList>
+          </Menu>
+          <Menu>
+            <MenuButton className="header__navigation-item menu__title">
```

Comment:
I happened to have my app in Polish while running this branch and noticed this menu looks a little weird when the text of the items is longer than they are in English.

Reply:
I also kind of dislike this from a UX perspective. I'd rather just put light/dark in settings, give Help first-class status, and lose the dropdown entirely if these are the only 3 items in it.

Reply:
I'm fine with moving help out of the menu, but I would really like to keep the dark mode toggle. YouTube, Twitter, and Reddit all have a dark mode toggle in their drop-down menu.